Watchman.Web/Areas/Messages/Models/Dtos/MessageDto.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using Watchman.DomainModel.Messages;
+
+namespace Watchman.Web.Areas.Messages.Models.Dtos
+{
+    public class MessageDto
+    {
+        public string Content { get; private set; }
+        public MessageUserDto User { get; private set; }
+        public MessageChannelDto Channel { get; private set; }
+        public MessageServerDto Server { get; private set; }
+        public DateTime SentAt { get; private set; }
+
+        public MessageDto()
```

Comment:
kiedy potrzebny jest pusty ctor?

Reply:
zapomniałem uzupełnić

Reply:
poprawione - ogólnie to obiekty DTO powinny mieć publiczne settery, żeby dało sie łatwo parsować na nie obiekty które dostajemy przez api 

czasami (albo zawsze, nie pamiętam) wymagany jest dodatkowo pusty konstruktor
jak nie ma żadnego to fizycznie istnieje pusty 
ale jak jest nasz, który ma parametry, to nie istnieje pusty